py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py

```diff
@@ -323,6 +410,15 @@ def __getitem__(
 
                 if isinstance(converters, (list, tuple)):
                     for candidate in converters:
+                        # If there are dynamic inputs but the converter doesn't support it explicitly, throw a warning.
+                        if (
```

Comment:
You could have a flag similar to experimental_decompositions that lets all converters through but default behavior is non dynamic converters should not be used in a dynamic context.

Reply:
Implemented it now